src/main/kotlin/nexters/admin/domain/attendance/AttendanceStatus.kt

```diff
@@ -1,9 +1,9 @@
 package nexters.admin.domain.attendance
 
-enum class AttendanceStatus(val value: String) {
-    PENDING("대기"),
-    ATTENDED("출석"),
-    TARDY("지각"),
-    UNAUTHORIZED_ABSENCE("무단결석"),
-    AUTHORIZED_ABSENCE("통보결석"),
+enum class AttendanceStatus(val value: String, val score: Int) {
+    PENDING("대기", 0),
+    ATTENDED("출석", 0),
+    TARDY("지각", -5),
+    UNAUTHORIZED_ABSENCE("무단결석", -15),
+    AUTHORIZED_ABSENCE("통보결석", -10),
```

Comment:
중요한건 아닌데, `score` 대신 `penaltyScore`는 어떤가요?

```suggestion
enum class AttendanceStatus(val value: String, val penaltyScore: Int) {
    PENDING("대기", 0),
    ATTENDED("출석", 0),
    TARDY("지각", 5),
    UNAUTHORIZED_ABSENCE("무단결석", 15),
    AUTHORIZED_ABSENCE("통보결석", 10),
```

Reply:
`score`라는 명칭이 출석 점수와 혼동되는 것 같네요! ResponseBody에서도 해당 속성명 `penaltyScore`로수정하겠습니다